deployment/Makefile

```diff
@@ -47,6 +47,13 @@ collectstatic:
 	# uwsgi container it seems so I use docker exec here
 	@docker exec -t -i $(PROJECT_ID)_uwsgi_1 python manage.py collectstatic --noinput
 
+reload:
```

Comment:
Nice - we should port this over to our other projects too so that it becomes a standard command...